content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -1,215 +1,146 @@
 ---
-Title: Cluster Recovery
-description:
+Title: Recovering a Failed Cluster
+description: 
 weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
-Cluster recovery in Redis Enterprise Software (RS) is used for restoring
-an entire cluster, usually due to a complete cluster failure. The
-recovery process is achieved by creating a new cluster that has the
-exact same configuration, databases, and data as the original cluster.
-
-At a high-level, the cluster recovery process consists of the following
-steps:
-
-1. Recovering the cluster configuration into the first node in the new
-    cluster.
-1. Recovering the rest of the nodes by joining nodes to the new cluster
-    that replace corresponding nodes in the old cluster.
-1. Recovering the databases, and recovering their data if data
-    persistence was configured (for additional details, refer to
-    [Database
-    persistence]({{< relref "/rs/concepts/data-access/persistence.md" >}}).
-
-The cluster recovery process is carried out by using the cluster
-configuration backup files and database persistence files that are
-created by the cluster. These files are stored in the persistence
-storage location (for additional details, refer to [Persistent and
-ephemeral
-storage]({{< relref "/rs/administering/designing-production/persistent-ephemeral-storage.md" >}}).
-
-Prerequisites:
-
-- You will need the persistent storage drives of the old cluster.
+When a cluster fails,
+you must recover the database with its previous cluster configuration, databases and data.
+To recover the cluster to the state that it was in before the failure,
+you must create a new cluster that has the same node configuration as the original cluster.
+
+Cluster recovery requires the cluster configuration backup files and database persistence files
```

Comment:
What's the difference between "backup files" and "persistence files"? I know what you mean, but I think the user won't understand

Reply:
Both the cluster configuration and the user data are both stored in "persistent backup files", I think the distinction should be between configuration data and user data. Also, we should be clear on the terminology, and I use R&D terminology. You'll want to use customer facing terminology, and that you will get better from the marketing guys